### PR TITLE
Added U.S. territories to feedback tool schema

### PR DESCRIPTION
**Pull request by ksantiagomorgan6**
## Description
Add the U.S. Territories to the state pick list

https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19283

## Testing done
Local testing completed

## Screenshots
Screen Shot 2019-11-25 at 1 48 56 PM

## Acceptance criteria
- [x] Feedback tool accept both US and US Territories.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/schemas/FEEDBACK-TOOL/schema.js [hunk 1]
```diff
@@ -1,407 +1,416 @@
...
       minLength: 1,
       maxLength: 255
     },
     state: {
-      type: 'string',
-      'enum': constants.states50AndDC.map(state => state.value), // backend accepts abbreviated state names
-      enumNames: constants.states50AndDC.map(state => state.label)
+      type: "string",
+      enum: constants.salesforceStates.USA.map(state => state.value), // backend accepts abbreviated state names
+      enumNames: constants.salesforceStates.USA.map(state => state.label)
```
Comment (shawkey-daniel) [inline src/schemas/FEEDBACK-TOOL/schema.js]: Here I was expecting constants.states.USA not salesForceStates.

Comment (ksantiagomorgan6) [reply src/schemas/FEEDBACK-TOOL/schema.js]: Updated it to constants.states.USA

**Comment by ksantiagomorgan6** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Comment by ksantiagomorgan6** [comment]
> It seems like a `package-lock.json` was committed. Was that intentional? We already have a lock file with `yarn.lock`, so I wonder if this might've be an artifact of mixing package managers.
> 
> The changes to `schema.js` don't seem necessary and seem to enforce a linting style that's inconsistent with `vets-website` and even the other schemas in this repo. Are there any significant changes in there that need to be retained? If it's entirely formatting, could that be rolled back?

Looks like those files where auto generated when running `npm run build` do you want't be to go ahead and uncommit those files?

**Comment by U-DON** [comment]
> > It seems like a `package-lock.json` was committed. Was that intentional? We already have a lock file with `yarn.lock`, so I wonder if this might've be an artifact of mixing package managers.
> > The changes to `schema.js` don't seem necessary and seem to enforce a linting style that's inconsistent with `vets-website` and even the other schemas in this repo. Are there any significant changes in there that need to be retained? If it's entirely formatting, could that be rolled back?
> 
> Looks like those files where auto generated when running `npm run build` do you want't be to go ahead and uncommit those files?

Yes, I think they should be reverted. Maybe a `yarn build` would avoid creating the `package-lock.json` and linting the schema.

**Comment by ksantiagomorgan6** [comment]
Yarn build also linted the feedback schema, but didn't produce the package-lock file

**Pull request merged**